query-service-impl/src/test/java/org/hypertrace/core/query/service/trino/QueryRequestToTrinoSQLConverterTest.java

```diff
@@ -217,6 +217,48 @@ void testQueryWithBooleanFilter() {
         executionContext);
   }
 
+  @Test
+  void testQueryWithIsTrinoFilter() {
+    QueryRequest queryRequest =
+        buildSimpleQueryWithFilter(createEqualsFilter("Event.isTrino", true));
+    TableDefinition tableDefinition = getDefaultTableDefinition();
+    defaultMockingForExecutionContext();
+
+    assertSQLQuery(
+        queryRequest,
+        "Select lower(to_hex(span_id)) FROM span-event-view WHERE "
+            + tableDefinition.getTenantIdColumn()
+            + " = '"
+            + TENANT_ID
+            + "'",
+        tableDefinition,
+        executionContext);
+  }
+
+  @Test
+  void testQueryWithMultipleFiltersAndIsTrinoFilter() {
+    Filter isEntryFilter = createEqualsFilter("Span.is_entry", true);
+    Filter isTrinoFilter = createEqualsFilter("Event.isTrino", true);
+    Filter isBareFilter = createEqualsFilter("Span.isBare", false);
+    QueryRequest queryRequest =
+        buildSimpleQueryWithFilter(
+            createCompositeFilter(Operator.AND, isEntryFilter, isTrinoFilter, isBareFilter)
```

Comment:
Can you add one more example of adding `isTrinoFilter` at beginning / end? 
Just to see if the converted SQL is not like:
```
AND ( AND is_bare = false )
```

Reply:
Will add